python/graphstorm/model/embed.py

```diff
@@ -168,6 +168,23 @@ class GSNodeEncoderInputLayer(GSNodeInputLayer):
         Number of layers of feedforward neural network for each node type in the input layers
     ffn_activation : callable
         The activation function for the feedforward neural networks.
+
+    Examples:
+    ----------
+    .. code:: python
```

Comment:
Please add a space line before the .. code:: python to let Spinix work properly.